internal/sdkmath/floor_go1.9.go

```diff
@@ -0,0 +1,93 @@
+// +build !go1.10
+
+package sdkmath
+
+import "unsafe"
+
+const (
+	uvone    = 0x3FF0000000000000
+	mask     = 0x7FF
+	shift    = 64 - 11 - 1
+	bias     = 1023
+	signMask = 1 << 63
+	fracMask = 1<<shift - 1
+)
+
+// Float64frombits returns the floating-point number corresponding
+// to the IEEE 754 binary representation b, with the sign bit of b
+// and the result in the same bit position.
+// Float64frombits(Float64bits(x)) == x.
+func Float64frombits(b uint64) float64 { return *(*float64)(unsafe.Pointer(&b)) }
```

Comment:
Are `Float64bits` and `Float64frombits` available in order versions?

Reply:
Yes the older versions have those functions